datalad_registry/overview.py

```diff
@@ -56,8 +55,8 @@ def overview():  # No type hints due to mypy#7187.
                 row[col] = getattr(info, col)
 
             ts = info.info_ts
-            if ts:
-                row["last_update"] = time.strftime("%Y-%m-%dT%H:%M%z", time.gmtime(ts))
+            if ts is not None:
```

Comment:
curious, why `if ts` isn't sufficient?

Reply:
`if ts is None` avoids ambiguity about whether a timestamp of zero seconds past the epoch is true or false.  (I seem to recall there being an issue with the `datetime` class regarding this in the past, but looking at the docs now, the only relevant bit I can find is that `time` values at midnight were false prior to Python 3.5.)

Reply:
good to know.  Thank you!